its/tests/src/test/java/its/ConnectedModeBackendTest.java

```diff
@@ -181,6 +187,174 @@ public void returnDescriptionSectionsForTaintRules() throws ExecutionException,
     }
   }
 
+  @Test
+  public void returnConvertedDescriptionSectionsForHotspotRules() throws ExecutionException, InterruptedException {
+    assumeThat(ORCHESTRATOR.getServer().version()).isGreaterThanOrEqualTo(Version.create("9.7"));
+
+    backend.getConfigurationService().didAddConfigurationScopes(new DidAddConfigurationScopesParams(
+      List.of(new ConfigurationScopeDto("project", null, true, "Project", new BindingConfigurationDto("ORCHESTRATOR", PROJECT_KEY_JAVA_HOTSPOT, false)))));
+
+    var activeRuleDetailsResponse = backend.getActiveRulesService().getActiveRuleDetails(new GetActiveRuleDetailsParams("project", javaRuleKey(ORCHESTRATOR, "S4792"))).get();
+
+    var description = activeRuleDetailsResponse.details().getDescription();
+    assertThat(description)
+      .extracting("right.introductionHtmlContent")
```

Comment:
I think using `extracting` here is not required. Would be cleaner with `assertThat(description.getRight().getIntroductionHtmlContent())`.